tasks/Task.cpp

```diff
@@ -57,18 +57,6 @@ void Task::processIO()
 
 }
 
-bool Task::setLed_list(std::vector<ledlamp_bowtech::LedLamp > const& ledlist)
-{
-    for (size_t i = 0; i < ledlist.size(); ++i)
-    {
-    	lamps.setPowerUpLightLevel(ledlist[i].power_up_light_level, ledlist[i].address);
-    	usleep(100000);
-    	lamps.setLightLevel(ledlist[i].light_level, ledlist[i].address);
-    	usleep(100000);
-    }
-
-    return(ledlamp_bowtech::TaskBase::setLed_list(ledlist));
-}
```

Comment:
Should I use an input port with just the std::vector of levels (i.e., std::vector of uint8_t) or with a vector of structs{address, level}?